Tests/Editor/TextureTests.cs

```diff
@@ -0,0 +1,57 @@
+using NUnit.Framework;
+using Unity.ProjectAuditor.Editor;
+using System.IO;
+using System.Text;
+using Unity.ProjectAuditor.EditorTests;
+using UnityEditor;
+using UnityEngine;
+
+
+namespace Unity.ProjectAuditor.EditorTests
+{
+    class TextureTests : TestFixtureBase
+    {
+        public int projectTextureCount;
+        public int resolution = 1;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            var texture = new Texture2D(resolution, resolution); //defaults: mipmaps = true & format = automatic
+            texture.SetPixel(0, 0, Random.ColorHSV());
+            texture.name = "ProceduralTextureForTest321.png";
+            texture.Apply();
+
+            byte[] bytes = texture.EncodeToPNG();
+
+            var temptesttexture = new TempAsset(texture.name, bytes);
+
+            var allTextures = AssetDatabase.FindAssets("t: Texture, a:assets");
+            projectTextureCount = allTextures.Length;
+            foreach (var foundTexture in allTextures)
+            {
+                var pathToTexture = AssetDatabase.GUIDToAssetPath(foundTexture);
+                if (pathToTexture.EndsWith("renderTexture")) { projectTextureCount -= 1; } }
+        }
+
+        [Test]
+        public void Texture_Properties_AreReported()
+        {
+            var TextureTests = Analyze(IssueCategory.Texture);
+
+            Assert.AreEqual(projectTextureCount, TextureTests.Length, "Checked Texture Count");
+
+            Assert.AreEqual("ProceduralTextureForTest321", TextureTests[0].customProperties[0], "Checked Texture Name");
+
+            Assert.AreEqual("Image", TextureTests[0].customProperties[2], "Checked TextureImporterType "); // Shown as "Default" in Editor but corresponds as "Image" (value returned) in API
+
+            Assert.AreEqual("AutomaticCompressed", TextureTests[0].customProperties[3], "Checked Texture Compression");
+
+            Assert.AreEqual("True", TextureTests[0].customProperties[5], "Checked MipMaps Enabled");
+
```

Comment:
please also check `customProperties[1]` and `customProperties[4]`, whatever those are.

Reply:
#1 successfully added to the Unit Tests. 
#4 is pending addition

Reply:
1 & 4 addition into the unit tests completed.